news/news.md

```diff
@@ -1,4 +1,7 @@
 # News
+## 03 September 2017
+* [Change Bank - Announcing Pre-ICO Details](change_03-09-17.md)
```

Comment:
Please, add this news as link and description ( add "## News" )  to project page